### DISCUSSION TITLE
[Bug]: Icons to indicate objects in the Events review screen are hard to see in Light mode

### DISCUSSION THREAD

**Discussion by supermarkert** [blakeblackshear/frigate#12811] [Report a Bug] [answered]
### Describe the problem you are having

The icons for object detection are white, on a very light gray background. With so little contrast, it's very difficult to see (especially on a larger screen when the icons are so small).  
Suggest using a darker color against the background using a color similar (but different) from the timestamp.  
This is only an issue in Light mode. Dark mode can be viewed fine.

![image](https://github.com/user-attachments/assets/0c4932e6-c12b-4899-b555-28742440432e)

### Steps to reproduce

View Event history for any camera

### Version

0.14.0-rc2

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

any

**Comment by hawkeye217** [comment, marked as answer]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/12812
